config.proto

```diff
@@ -0,0 +1,75 @@
+syntax = "proto3";
+
+package release;
+
+service Release {
+    // Get information about a release from its id
+    rpc GetRelease(string) returns (Release) {};
+    // List all releases matching the request
+    rpc ListReleases (ListReleasesRequest) returns (ListReleasesReponse) {};
```

Comment:
Generally List operation returns a list of shallow objects instead of everything in the release to optimize bandwidth and load time.

Reply:
done